ui/app/templates/components/http-requests-table.hbs

```diff
@@ -0,0 +1,27 @@
+<table class="is-fullwidth">
```

Comment:
There's a few things to make a11y better on tables - using a `<caption>` element and the `scope` attribute on any `th` elements. The latter is more important IIRC - this is a good article on it https://a11yproject.com/posts/accessible-data-tables/

Reply:
ah yes, this is super helpful. thank you for the reminder and information!